clients/src/main/java/org/apache/kafka/clients/producer/KafkaProducer.java

```diff
@@ -96,7 +96,7 @@
  * Properties props = new Properties();
  * props.put("bootstrap.servers", "localhost:9092");
  * props.put("acks", "all");
- * props.put("delivery.timeout.ms", 30000);
+ * props.put("delivery.timeout.ms", 30001);
```

Comment:
I think we should actually make this example simpler. Most of these configs don't need to be changed. We could change the example to be something like:

```java
 * Properties props = new Properties();
 * props.put("bootstrap.servers", "localhost:9092");
 * props.put("acks", "all");
 *
 * try (Producer<String, String> producer = new KafkaProducer<>(props, new StringSerializer(), new StringSerializer())) {
 *     for (int i = 0; i < 100; i++)
 *         producer.send(new ProducerRecord<String, String>("my-topic", Integer.toString(i), Integer.toString(i)));
 * }
```

What do you think?

Reply:
Okay, I'll update it.